### ISSUE TITLE
Signature plot

### ISSUE THREAD

**Issue by pkolokoussis** [semiautomaticgit/SemiAutomaticClassificationPlugin#19]
Hi Luca,
First of all thank you for the very good work you have done. Since, last year I have started to teach remote sensing procedures using your plugin and giving up proprietary software. With the help of our students we will try to help you in the translation of the plugin, although I personally do prefer to use all programs in english.
One of my students came today and reported that the signature plot does not work correctly after the last update of the plugin. I verified this information and indeed the scaling of the X axis of the graph is not correct. Could you please have a look at this issue?
Thank you in advance

**Comment by semiautomaticgit** [comment]
Hi, and thank you for your willing to help in the translation. I also prefer the interface in English, you could consider translating the user manual to your language.

About your issue, could you describe what images are you using. The image should be converted to reflectance for proper display, or (in case the image has pixel values in the order of thousands) edit the Multiplicative factor in the Band set to have decimal pixel values http://semiautomaticclassificationmanual-v5.readthedocs.io/en/latest/main_interface_window.html#band-set

**Comment by pkolokoussis** [comment]
Hi Luca, 
Thank you for your answer. Because this is an entry course in digital remote sensing we tried to avoid atmospheric corrections for the time being and we used Landsat 4-5 images with integer brightness values. I tried the proposed solution with the multiplicative factor and it works, but the plot used to work with brightness values too. Anyway, we will do it this way. 
We will let you know as soon as we have news about the translation of the manual, which will be very useful for our students anyway.
